clang/test/Driver/sycl-native-cpu.cpp

```diff
@@ -19,8 +19,11 @@
 // CHECK-WIN-NOT: dwarf
 
 // checks that -sycl-opt is not enabled by default on NativeCPU so that the full llvm optimization is enabled
+// Also check that we pass the expected backend uptions.
 // RUN:   %clang -fsycl -fsycl-targets=native_cpu -### %s 2>&1 | FileCheck -check-prefix=CHECK-OPTS %s
 // CHECK-OPTS-NOT: -sycl-opt
+// CHECK-OPTS: "-Wno-override-module" "-mllvm" "-sycl-native-cpu-backend"
```

Comment:
```suggestion
// CHECK-OPTS: clang{{.*}} "-triple" "native_cpu"
// CHECK-OPTS-SAME: "-Wno-override-module" "-mllvm" "-sycl-native-cpu-backend"
```
Just a small adjustment to verify where the opts are being added.

Reply:
Thanks! That isn't the line where those options appear, so that check would fail, but good idea, so I've adjusted it so that it checks the correct line.